docker-compose.yml

```diff
@@ -32,6 +32,56 @@ services:
   rosie:
     image: serenata/rosie
 
+  prometheus:
+    image: arthursens/serenata-prometheus
```

Comment:
arthursens/serenata-prometheus image was built from [this Dockerfile](https://github.com/okfn-brasil/serenata-de-amor/pull/484/files#diff-831949f2ad1dcdf050d81cf5a100bc7eR1)